### DISCUSSION TITLE
Release v1.6.5

### DISCUSSION THREAD

**Discussion by bao-cn** [6get-xiaofan/DarkMode#17] [Ideas]
此版本修复了一些Bug，新增一些功能优化一些问题

1. 修复运行时突然的异常弹窗；
2. 尝试修复壁纸无法切换问题；
3. 新增自定义应用模式；
4. 新增启动软件时的通知提醒；
5. 优化字体，窗口大小；
6. 一些细节优化。

<hr /><em>This discussion was created from the release <a href='https://github.com/Melon-Studio/DarkMode/releases/tag/v1.6.5'>Release v1.6.5</a>.</em>

**Comment by atrolley** [comment]
你好，我选择壁纸 ，壁纸还是切换失败。系统版本是21H2

**Reply by bao-cn** [reply]
此问题将于未来发布的2.0版本修复
